serviceWrappers/linux/kLiveController

```diff
@@ -93,28 +105,28 @@ restart() {
     start
 }
 
-status() {
+###
+# check whether process is running based on /proc/[pid]/exe
+###
+my_status() {
     RETVAL=0
     for prefix in 0 1 ; do
-        PIDFILE="$PID_DIR/${NAME}_${prefix}.pid"
-        if [ -f $PIDFILE ] ; then
-	    VAL=$( ( cat ${PIDFILE} | xargs ps -p ) | wc -l )
-	    if [ $VAL -gt 1 ] ; then
-            echo "${NAME}_${prefix} is running with PID $( cat $PIDFILE )"
-            else
-            echo "${NAME}_${prefix} is not running but there is stale pidFile: $PIDFILE"
-            RETVAL=$(( $RETVAL + 1 ))
- 	    fi
-        else
-            echo "${NAME}_${prefix} is not running."
-	        RETVAL=$(( $RETVAL + 1 ))
+        PIDFILE="${PID_DIR}/${NAME}_${prefix}.pid"
+        if [ -r ${PIDFILE} ]; then
+            status -p ${PIDFILE}
+	        if [ $? -ne 0 ]; then
+	            RETVAL=$?
+	            return ${RETVAL}
+	        fi
+	    else
+	        echo "${PIDFILE} is empty, ${NAME} is not running (verify by: ps aux | grep node)"
```

Comment:
need to properly indent, also, the message is wrong. This point in the code will only be reached if PIDFILE is not readable, not if it's empty.